Spec/RealtimeClientPresence.swift

```diff
@@ -136,6 +136,59 @@ class RealtimeClientPresence: QuickSpec {
                 }
             }
 
+            // RTP16
+            context("Connection state conditions") {
+
+                // RTP16c
+                pending("should result in an error if the connection state is INITIALIZED") {
```

Comment:
Yes, you right. The connection is in the `CONNECTING` state. I verified the ruby test (keep in mind that I'm not comfortable with ruby) and it seems to check the `INITIALIZED` state: https://github.com/ably/ably-ruby/blob/6e8e1ea7949026f19fa4c1405ccb5d8b96c3aeb9/spec/acceptance/realtime/presence_spec.rb#L58-L64